core/src/main/java/io/grpc/ClientCallImpl.java

```diff
@@ -85,27 +108,21 @@
     ClientTransport get();
   }
 
+  private enum State {INIT, STARTED, HALF_CLOSED, CANCELED}
```

Comment:
I had seen it, but  was under the impression that it was for Channels, not Calls.  I had intended this enum to just be for managing internal state, rather than using multiple booleans.